.github/workflows/alerts.yaml

```diff
@@ -0,0 +1,29 @@
+name: Test alerts
+
+on:
+  push
+
+jobs:
+  run-alert-tests:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        tests:
+          - name: 1 minute interval tests
+            file: monitoring/alerts.test.yaml
+          
+          - name: 10 seconds interval tests
+            file: monitoring/alerts.10s.test.yaml
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Render and test ${{ matrix.tests.name }}
+        uses: scality/action-prom-render-test@1.0.1
+        with:
+          alert_file_path: monitoring/alerts.yaml
+          test_file_path: ${{ matrix.tests.file }}
+          alert_inputs: >-
+            namespace=zenko,service=artesca-data-connector-s3api-metrics,replicas=3,systemErrorsWarningThreshold=0.03,systemErrorsCriticalThreshold=0.05,listingLatencyWarningThreshold=0.300,listingLatencyCriticalThreshold=0.500,deleteLatencyWarningThreshold=0.500,deleteLatencyCriticalThreshold=1.000
```

Comment:
should not need to specify all inputs : just `namespace`, `service` and `replicas` should be OK.

Reply:
```suggestion
            namespace=zenko,service=artesca-data-connector-s3api-metrics,replicas=3
```

Reply:
done.